storage/cache/subtree_cache_test.go

```diff
@@ -132,10 +137,10 @@ func TestCacheFlush(t *testing.T) {
 	m := NewMockNodeStorage(mockCtrl)
 	c := NewSubtreeCache(defaultLogStrata, populateLogSubtreeNodes(rfc6962.DefaultHasher), prepareLogSubtreeWrite())
 
-	h := "01234567"
-	nodeID := tree.NewNodeIDFromHash([]byte(h))
+	id := compact.NewNodeID(0, 12345)
+	nodeID, _ := tree.NewNodeIDForTreeCoords(int64(id.Level), int64(id.Index), maxLogDepth)
```

Comment:
Are any significant errors being ignored here?

Reply:
Most certainly not, but I brought back the `MustCreateNodeIDForTreeCoords` while it's needed in this test.